app/controllers/album.js

```diff
@@ -0,0 +1,36 @@
+const axios = require('axios');
+const logger = require('../../app/logger');
+const PREVIOUS_MESSAGE = 'Making request..';
+const MESSAGE_OK = 'Request made correctly, sending ...';
+
+const getAlbums = async (req, res) => {
+  const urlAlbums = 'https://jsonplaceholder.typicode.com/albums';
```

Comment:
Use env var for this. Check config file.

Reply:
We put urls and secrets always as env vars